Source/AIFlatSwitch.swift

```diff
@@ -176,6 +176,7 @@ import UIKit
 		// Animate to new state
 		if animated {
 			self.addAnimations(desiredSelectedState: isSelectedInternal)
+            self.accessibilityValue = isSelected ? "checked" : "unchecked"
```

Comment:
@marcuswu I also noticed indentation looks weird here, is it same for you?

Reply:
Fixed -- I had spaces vs the rest of the file which uses tabs. It appeared the same in my editor.